launch_yaml/launch_yaml/parser.py

```diff
@@ -51,3 +53,8 @@ def load(
         finally:
             if didopen:
                 fileobj.close()
+
+    @classmethod
+    def get_file_extensions(cls) -> Set[Text]:
+        """Return a set of file extensions for this parser."""
```

Comment:
@christophebedard nit: 
```suggestion
        """Return the set of file extensions known to this parser."""
```

Reply:
changed in bb5538ffc4ed9cc5bfbf9d78a999d3a29c79a688 (I also applied this change for the launch_xml parser)